tests/FakeItEasy.Specs/WrappingFakeSpecs.cs

```diff
@@ -306,6 +330,12 @@ public void OutAndRefMethod(ref int @ref, out int @out)
                 @ref += 1;
                 @out = 42;
             }
+
+            public event EventHandler? SomeEvent;
+
+#pragma warning disable CA1030 // Use events where appropriate
+            public void RaiseSomeEvent() => this.SomeEvent?.Invoke(this, EventArgs.Empty);
+#pragma warning restore CA1030 // Use events where appropriate

```

Comment:
IMO this CA1030 warning is useless. If I wanted this to be an event, I would have made it an event (which, in that case, wouldn't make any sense). We might want to disable this warning completely (we already had a few suppressions in the VB integration tests)

Reply:
Sounds good. I could support it.